script.js

```diff
@@ -0,0 +1,28 @@
+function daysInYear(year) {
+  try {
+    if(!daysAmountInYear(year)) {
+      throw new TypeError("exeption");
+    } 
+      return daysAmountInYear(year);
+    } catch (error) {
+      return error.message;
```

Comment:
Based on the requirements `it should throw an exception`. This line doesn't throw an exception.